### DISCUSSION TITLE
New store online : Grandes Marques

### DISCUSSION THREAD

**Discussion by JBaudeau** [saleor/saleor#11965] [Show and tell]
Hi all,

We're proud to present you with a new store powered by Saleor, Strapi and Gatsby: Grandes Marques ([https://www.grandes-marques.fr](https://www.grandes-marques.fr))!

Grandes Marques is a company selling high quality household linen products, based in Toulouse, France.

We used the open-source versions of Saleor, Strapi and Gatsby, all hosted on Google Cloud Platform, to build a very powerful and SEO optimized E-commerce website.

You can see a LinkedIn post about it [here](https://www.linkedin.com/posts/innovationdigitale_ecommerce-saleor-ventes-activity-7029408750654345216-CoQs) (don't be afraid of the French language 😄).

We would be glad hearing what you think about it!

Jolyan Baudeau, on behalf of Coperbee ([https://www.coperbee.fr/](https://www.coperbee.fr/))

**Comment by karolkielecki** [comment]
Great work @JBaudeau! 👏

**Comment by peelar** [comment]
Hello @JBaudeau 👋 

Congrats on an awesome shop! I hope you got to try out the linen, it looks super comfy 😁.

I would love to hear your impressions about Saleor, especially the things that could have made your job easier 🧐. If you have time, please shoot me an email at adrian.pilarczyk@saleor.io, and I will respond with a couple of questions 🙇‍♂️.

**Comment by timuric** [comment]
@JBaudeau congratulations on the launch! I am quite curious to know what was your experience working with Gatsby
